snap/gui/sidekick.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=0.6
```

Comment:
This file seems to be a duplicate do we need both or is one of them a past artifact from the configuration?

Reply:
No caso um seria para o AppImage e outro para o Snap, pois o snap tem uma configuração um pouco diferente